PSMeraki/Public/Get-MrkOrgEndpoint.ps1

```diff
@@ -10,10 +10,10 @@ function Get-MrkOrgEndpoint {
     [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
     if (!$orgBaseUri){
         Write-Verbose "Get-MrkOrgEndpoint: global orgBaseUri is empty, retrieving organization specific meraki endpoint"
-        $orgURI = 'https://api.meraki.com/api/v0/organizations'
+        $orgURI = 'https://api.meraki.com/api/v1/organizations'
```

Comment:
The step from v0 to v1 of Meraki's rest API has consequences for some cmdlets, they will no longer work in v1.
https://developer.cisco.com/meraki/whats-new/#!v1-0-0
Will the improvements you made, work on /v0/ too?